pkg/network/tracer/tracer.go

```diff
@@ -837,6 +839,17 @@ func (t *Tracer) DebugDumpProcessCache(ctx context.Context) (interface{}, error)
 	return nil, nil
 }
 
+func (t *Tracer) verifyPESIsSupported(kernelVersion *ebpfkernel.Version) {
+	if !tr.cfg.EnableProcessEventMonitoring {
+		return
+	}
+	if kernelVersion.IsRH7Kernel() || (!kernelVersion.IsRH8Kernel() && kernelVersion.Code < ebpfkernel.Kernel4_15) {
```

Comment:
I wouldn't rely upon these helpers, because they count on `/etc/os-release` being correctly mounted into the container. Do you know what the actual kernel requirements are for the feature?

Reply:
I am basing these off of this check:
https://github.com/DataDog/datadog-agent/blob/main/pkg/security/probe/probe_ebpf.go#L192-L197

Reply:
So that check is `RH7 || RH8 || kernel >= 4.15` is OK

Reply:
To be clear, your logic is incorrect and doesn't match the check you are basing it on, unless you want different requirements?

Reply:
https://github.com/DataDog/datadog-agent/blob/main/pkg/security/probe/probe_ebpf.go#L178-L181

there's another check for RH7 elsewhere